src/styles/block/page.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  margin: 0;
+  padding: 0;
```

Comment:
`html` tag doesn't have default `padding` and `margin`, delete `padding`, move `margin` to body